GiniCapture/Classes/Core/Screens/Help/HelpMenuViewController.swift

```diff
@@ -10,90 +10,63 @@ import UIKit
 
 /**
  The `HelpMenuViewControllerDelegate` protocol defines methods that allow you to handle table item selection actions.
- 
+
  - note: Component API only.
  */
 
 public protocol HelpMenuViewControllerDelegate: AnyObject {
-    func help(_ menuViewController: HelpMenuViewController, didSelect item: HelpMenuViewController.Item)
+    func help(_ menuViewController: HelpMenuViewController, didSelect item: HelpMenuViewController.MenuItem)
 }
 
 /**
  The `HelpMenuViewController` provides explanations on how to take better pictures, how to
- use the _Open with_ feature and which formats are supported by the Gini Capture SDK. 
+ use the _Open with_ feature and which formats are supported by the Gini Capture SDK.
  */
 
-final public class HelpMenuViewController: UITableViewController {
-    
+public final class HelpMenuViewController: UITableViewController {
     public weak var delegate: HelpMenuViewControllerDelegate?
     let giniConfiguration: GiniConfiguration
     let tableRowHeight: CGFloat = 64
     var helpMenuCellIdentifier = "helpMenuCellIdentifier"
-    
-    public enum Item {
+
+    public enum ItemPresentationMode {
         case noResultsTips
-        case openWithTutorial
-        case supportedFormats
-        
-        var title: String {
-            switch self {
-            case .noResultsTips:
-                return .localized(resource: HelpStrings.menuFirstItemText)
-            case .openWithTutorial:
-                return .localized(resource: HelpStrings.menuSecondItemText)
-            case .supportedFormats:
-                return .localized(resource: HelpStrings.menuThirdItemText)
-            }
-        }
-        
-        var viewController: UIViewController {
-            let viewController: UIViewController
-            switch self {
-            case .noResultsTips:
-                let title: String = .localized(resource: ImageAnalysisNoResultsStrings.titleText)
-                let topViewText: String = .localized(resource: ImageAnalysisNoResultsStrings.warningHelpMenuText)
-                viewController = ImageAnalysisNoResultsViewController(title: title,
-                                                                      subHeaderText: nil,
-                                                                      topViewText: topViewText,
-                                                                      topViewIcon: nil)
-            case .openWithTutorial:
-                viewController = OpenWithTutorialViewController()
-            case .supportedFormats:
-                viewController = SupportedFormatsViewController()
-            }
-            
-            return viewController
-            
-        }
+        case helpItem
```

Comment:
We add the `case custom(String, UIViewController)` to `Item` here.

Then in `var title: String {` we add:
```
case .custom(let title, _):
    return title
```

And in `var viewController: UIViewController {`:
```
case .custom(_, let viewController):
    return viewController
```

Then we rename `lazy var items: [Item] = {` to `lazy var defaultItems: [Item] = {` to make it clear that it doesn't contain custom items.

Continued below ->